app-backend/tile/src/main/scala/image/Mosaic.scala

```diff
@@ -267,8 +267,10 @@ object Mosaic {
           val tiles = maybeTiles.flatten
           if (tiles.nonEmpty)
             Option(tiles.reduce(_ merge _))
-          else
-            Option.empty[MultibandTile]
+          else {
+            val cellType = IntUserDefinedNoDataCellType(0)
+            Option(ArrayMultibandTile.empty(cellType, 3, 256, 256))
+          }
```

Comment:
This is a bit inefficient, since the empty tile still goes through the `renderPng` call, which has a non-trival computation task (even for empty tiles). 

What you could do instead is have a `val` in MosaicRoutes:

```scala
val emptyTilePng = IntArrayTile.alloc(256, 256).renderPng // Should constants be configurable here?
```

Here notice I use `alloc` instead of `empty`; `empty` for the default cell type (`IntConstantNoDataCellType`) would put in `NODATA` values (which are `Int.MinValue`); `alloc` will simply initialize to default value of that primitive type (for `Ints` it's 0). 

Notice I also use `renderPng` (without params) on a single band tile. Since really all we would care about is the rendered 256x256 PNG, it doesn't matter that it comes from a single band instead of a multiband. Also since we want all of our colors to be literally `0x00000000`, we can use this overload of renderPng: https://geotrellis.github.io/scaladocs/latest/index.html#geotrellis.raster.render.PngRenderMethods@renderPng():geotrellis.raster.render.Png

We then have a captured empty PNG that we can send back from our service routes:

```scala
pathPrefix("export") {
        parameter("bbox".?, "zoom".as[Int]?) { (bbox, zoom) =>
          get {
            complete {
              Mosaic.render(projectId, zoom, bbox).map { maybeRender =>
                pngAsHttpResponse {
                    maybeRender.map(_.renderPng()).getOrElse(emptyTilePng)
                }
              }
            }
          }
        }
      }
```

Notice the slight refactor, taking the `pngAsHttpResponse` out of the option map, and using getOrElse.

That last bit could be written in a couple of different ways; another option is

`maybeRender.fold(emptyTilePng)(_.renderPng())`